components/faq.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+    </head>
+    <body>
+        <main>
+        Need Help?
+            <input type="search" name="searchFaq">
+       
+        <h1>FAQS</h1>
+        <section class="question">
+            Question 1
+        
+        <div class="response">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+            Integer odio elit, placerat ut varius in, porttitor nec eros.
+             Mauris eros urna, mollis sit amet erat a, luctus rhoncus neque. 
+        </div>
+    </section>
+        <section class="question">
+            Question 2
+        
+        <div class="response">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+            Integer odio elit, placerat ut varius in, porttitor nec eros.
+             Mauris eros urna, mollis sit amet erat a, luctus rhoncus neque. 
+        </div>
+    </section>
+        <section class="question">
+            Question 3
+        
+        <div class="response">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+            Integer odio elit, placerat ut varius in, porttitor nec eros.
+             Mauris eros urna, mollis sit amet erat a, luctus rhoncus neque. 
+        </div>
+    </section>
+        <section class="question">
+            Question 4
+        
+        <div class="response">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+            Integer odio elit, placerat ut varius in, porttitor nec eros.
+             Mauris eros urna, mollis sit amet erat a, luctus rhoncus neque. 
+        </div>
+    </section>
+</main>
```

Comment:
Use Prettier!!! All closing tags have to be at the same indentation level as the opening ones.